DEVELOPMENT.md

```diff
@@ -15,13 +15,13 @@
 ## Running the Plugin
 
 1.  Clone [googleapis](https://github.com/googleapis/googleapis) and
-    [gapic-showcase](https://github.com/googleapis/gapic-showcase/) and install
-    protoc.
+    [gapic-showcase](https://github.com/googleapis/gapic-showcase/).
 
 2.  Copy the protos from Showcase into googleapis/google/showcase.
 
     ```sh
-    cp gapic-showcase/schema/google/showcase/v1beta1 googleapis/google/showcase/v1beta
+    mkdir googleapis/google/showcase
+    cp -r gapic-showcase/schema/google/showcase/v1beta1 googleapis/google/showcase/v1beta1
     ```
 
 3.  Add the new microgenerator rules to the protobuf directory's `BUILD.bazel`
```

Comment:
I think it's better to clarify which file needs to be modified.
```suggestion
3.  Add the new microgenerator rules to `googleapis/google/showcase/v1beta/BUILD.bazel`
```

Reply:
Good idea. Note it's `v1beta1`, not `v1beta`, which is what I'm fixing in this PR too.

Reply:
Right, sorry, fixed.